packages/eslint-plugin-pf-codemods/src/rules/helpers/nodeIsComponentNamed.ts

```diff
@@ -0,0 +1,9 @@
+import { JSXElement } from "estree-jsx";
+
+export function nodeIsComponentNamed(node: JSXElement, componentName: string) {
```

Comment:
@adamviktora seems like you and I built similar helpers (this from you, `checkMatchingJSXOpeningElement` in the nodeMatches file from me)? Wdyt about us combining them into a single rule + if so, any preference on naming?

Reply:
We can keep yours `checkMatchingJSXOpeningElement`, it is more general and we already have it as part of the initial general rule. I can refactor the two rules that are using this `nodeIsComponentNamed` to use `checkMatchingJSXOpeningElement` and then delete this helper.